TaskClient.cs

```diff
@@ -11,8 +11,8 @@ namespace Gofer.NET
     public class TaskClient
     {
         private static readonly object Locker = new object();
-        
-        private const int PollDelay = 100;
+
+        public int PollDelay { get; set; } = 100;
```

Comment:
Leaving a configuration value of the TaskClient in a settable property sets off my code spidey sense.

I'd prefer to have a pollDelay parameter (with a default value of 100) in the constructor that sets a private field _pollDelay.